tests/gtest/avifreadimagetest.cc

```diff
@@ -88,6 +89,177 @@ TEST(PngTest, RgbColorTypeWithTrnsBeforePlte) {
   EXPECT_EQ(image->alphaPlane, nullptr);
 }
 
+constexpr size_t kColorProfileSize = 376;
+constexpr size_t kGrayProfileSize = 275;
+
+// Verify we can read a color PNG file tagged as gamma 2.2 through gAMA chunk,
+// and set transfer characteristics correctly.
+TEST(PngTest, ColorGamma22) {
+  const auto image = testutil::ReadImage(data_path, "ffffcc-gamma2.2.png");
+  ASSERT_NE(image, nullptr);
+
+  // gamma 2.2 should match BT470M
+  EXPECT_EQ(image->transferCharacteristics,
+            AVIF_TRANSFER_CHARACTERISTICS_BT470M);
+
+  // should not generate ICC profile
+  EXPECT_EQ(image->icc.size, 0u);
+}
+
+// Verify we can read a color PNG file tagged as gamma 1.6 through gAMA chunk,
+// and generate a color profile for it.
+TEST(PngTest, ColorGamma16) {
+  const auto image = testutil::ReadImage(data_path, "ffffcc-gamma1.6.png");
+  ASSERT_NE(image, nullptr);
+
+  // if ICC profile generated, CP and TC should be set to unspecified
+  EXPECT_EQ(image->colorPrimaries, AVIF_COLOR_PRIMARIES_UNSPECIFIED);
+  EXPECT_EQ(image->transferCharacteristics,
+            AVIF_TRANSFER_CHARACTERISTICS_UNSPECIFIED);
+
+  // should generate a color profile
+  EXPECT_EQ(image->icc.size, kColorProfileSize);
+
+  // Test of generated profile is done in test_cmd_icc_profile
```

Comment:
Nit: Generated profile is tested in test_cmd_icc_profile.sh

Fix all three occurrences of this comment.

Reply:
Done